coordinator/statement_executor.go

```diff
@@ -932,6 +934,98 @@ func (e *StatementExecutor) executeShowSubscriptionsStatement(stmt *influxql.Sho
 	return rows, nil
 }
 
+func (e *StatementExecutor) executeShowTagKeys(q *influxql.ShowTagKeysStatement, ctx *query.ExecutionContext) error {
+	var dis []meta.DatabaseInfo
+	if q.Database != "" {
```

Comment:
I don't think that `q.Database` can be empty. The `StatementRewriter` should have set it I think. Did you encounter a case where it was empty?

Reply:
I think you're right. Not sure where I was thinking it could be empty. Fixed in 156f25a.